src/controllers/blocks/BlocksExtrinsicsController.ts

```diff
@@ -0,0 +1,68 @@
+import { ApiPromise } from '@polkadot/api';
+import { RequestHandler } from 'express';
+
+import { BlocksService } from '../../services';
+import { INumberParam } from '../../types/requests';
+import AbstractController from '../AbstractController';
+
+interface ControllerOptions {
+	finalizes: boolean;
+}
+
+export default class BlocksExtrinsicsController extends AbstractController<BlocksService> {
+	constructor(api: ApiPromise, private readonly options: ControllerOptions) {
+		super(api, '/blocks/:blockId/extrinsics', new BlocksService(api));
+		this.initRoutes();
+	}
+
+	protected initRoutes(): void {
+		this.safeMountAsyncGetHandlers([
+			['/:extrinsicsIndex', this.getExtrinsicsByExtrsinsicIndex],
```

Comment:
Should we change all mentions of `extrinsicsIndex` to `extriniscIndex` since we only get a single extrinsic?

Reply:
I think so yes; one could argue that it is one index (singular) within a collection of extrinsics and so `extrinsicsIndex` isn't wrong. To my ears `extrinsicIndex` sounds better though.

Reply:
Yea I agree with you both. I actually had to say it out loud a few times but `extrinsicIndex` definitely makes it sound way clearer.